sessionctx/variable/session.go

```diff
@@ -1553,9 +1543,6 @@ func (s *SessionVars) GetSystemVar(name string) (string, bool) {
 	} else if name == ErrorCount {
 		return strconv.Itoa(int(s.SysErrorCount)), true
 	}
-	if name == TiDBSlowLogMasking {
-		name = TiDBRedactLog
-	}
```

Comment:
This variable was previously removed, so this code path is not reached. It was detected when the `TiDBSlowLogMasking` was no longer exported.